nestris_ocr/scan_strat/scan_helpers.py

```diff
@@ -11,13 +11,19 @@
 
 
 PATTERNS = {
-    "score": "ADDDDD",
+    "score": "DDDDDD",
     "lines": "DDD",
-    "level": "AA",
+    "level": "TD",
     "stats": "DDD",
-    "das": "BD",
+    "das":   "BD",
```

Comment:
I believe this is what `black` didn't like 😂

Reply:
Oops 😓 . I pushed with `--no-verify` after my local precommit failed, and it was late I got the message:
```
env: python3.7: No such file or directory
```

I'll try to fix that today

Reply:
Oops 😓 . I pushed with `--no-verify` after my local precommit failed, and it was late I got the message:
```
env: python3.7: No such file or directory
```

I'll try to fix that today

Reply:
And fixed! Next PRS won't fail on lint :)